modules/luci-mod-network/htdocs/luci-static/resources/view/network/interfaces.js

```diff
@@ -1595,9 +1595,19 @@ return view.extend({
 			_('This prefix is randomly generated at first install.'));
 		o.datatype = 'cidr6';
 
-		o = s.option(form.Flag, 'packet_steering', _('Packet Steering'), _('Enable packet steering across all CPUs. May help or hinder network speed.'));
+		o = s.option(form.ListValue, 'packet_steering', _('Packet Steering'), _('Enable packet steering across all CPUs. May help or hinder network speed.'));
+		o.value('', _('Disabled'));
+		o.value('1',_('Enabled'));
+		o.value('2',_('Enabled - (old style)'));
 		o.optional = true;
-
+				
+		var steer_flow = uci.get('network', 'globals', 'steering_flows');	
+		
+		o = s.option(form.Value, 'steering_flows', _('Steering flows'), _('Steer up to 128 locally received flows per device to the CPU receiving them.'));
```

Comment:
`to the CPU receiving them` needs re-wording.

Reply:
`to the core who is receiving`
 Better or what do you suggest? Thank you

Reply:
Consider: 

I send this to you who is (already) receiving them (anyway).

It's effectively already said. Do you have a source that describes this? I could make a suggestion.

Reply:
Thanks for suggestion, so, the function is ```
queues = glob(`/sys/class/net/${dev}/queues/rx-*/rps_flow_cnt`);
	for (let queue in queues) {
		if (debug || do_nothing)
			warn(`echo ${local_flows} > ${queue}\n`);
		if (!do_nothing)
			writefile(queue, `${local_flows}`);
	}
```
With the new script we can set the rps.
To understand what Felix do I have read this https://access.redhat.com/documentation/it-it/red_hat_enterprise_linux/7/html/performance_tuning_guide/sect-red_hat_enterprise_linux-performance_tuning_guide-networking-configuration_tools at RPS section 9.3.6.

Thanks again.

Reply:
First line:

used to direct packets to specific CPUs for processing

So something like:

steers up to 128 packet flows per device to specific CPUs for processing 

Does that capture the essence?
Is there some reason it's not named rps here? Steering is more generalized nowadays with multiple options for it as you see at the link.

Reply:
If we're going to use the RPS acronym, then it needs explaining (it's not common vernacular). Have a look through the code-base and see how the mouse-overs for acronyms are handled.

e.g. `<abbr title=\"Domain Name System\">DNS</abbr>`

Reply:
Done, thanks